docs/evolve_spec.rst

```diff
@@ -49,7 +49,7 @@ Backwards Incompatible Changes
   * An old receiver may have application-layer dependencies on the field
     type. In statically typed languages deserialization will fail.
 
-* Adding a new tag to a union without a `catch-all tag <lang_ref.rst#union-catch-all>`_.
+* Adding a new tag to a closed union.
```

Comment:
Nit: Let's be consistent about "a new tag" and "another tag" between here and the next part. I think "a new tag" is slightly better.